multinode_integration_tests/src/utils.rs

```diff
@@ -138,6 +138,7 @@ impl From<&dyn MASQNode> for AccessibleGossipRecord {
                 accepts_connections: masq_node.accepts_connections(),
                 routes_data: masq_node.routes_data(),
                 version: 0,
+                country_code: "".to_string(),
```

Comment:
Shouldn't this always be some letters? Like "ZZ" at least. But as well as that, I wonder why you didn't add a new method to the trait `MASQNode` which would return the country code of that Node. 
Therefore here we want to read that as if from inside the Node by using a getter method as you can see those others up here.   

This has led me to a question, how would you prepare (or mock up) an exact country code for a Node in the test, because that's not a command line parameter, right? The only way, now, it can get set is to derive it from the ip provided or detected at this Node. However, we cannot just joggle with the ip, set by the inputs, because we need it to stay the correct one, set for the whole Docker container when started from an image. So, I think, we actually cannot have the Multinode test Real Nodes be using such a country code of a free choice. It will ever be a narrow range of IPs, the same one as we have to distinguish between every container with a Node. 

Did you consider what country code it generates if you go and look at the standard IPs used by these controlled Nodes in the tests? It might be interesting to know. What country is it? Is there still a way to give it a special country code taking place only in multinode tests, similarly to how we work a special chain designed only for this environment. It may not doesn't make any sense here. I'm just sprinkling ideas so that you can reach new perspectives. Anyway, knowing what country code matches the containers' IPs might be good knowledge.